lib/sequenceserver.rb

```diff
@@ -208,6 +208,10 @@ def using_ssh?
       true if ENV['SSH_CLIENT'] || ENV['SSH_TTY'] || ENV['SSH_CONNECTION']
     end
 
+    def compatible_linux_with_gui?
```

Comment:
Call it `has_xdg?`. There would be no point determining the presence of xdg-open command if the DISPLAY not set.